### PR TITLE
Feat/add pop up window

### LINKED ISSUE #213: [FRONTEND] Add pop-up window if position is already created
🚨 **Attention: Please read the guidelines before applying; otherwise, you will be ignored.** 🚫📋
0. Please, join the telegram group to keep in touch https://t.me/+m0IGRiBvLtsyMzIy 
1. Read guideline [here](https://gist.github.com/djeck1432/f722f5fe518c80b248ff609627da25e5)
2. Create new pop-up window follow the figma (text me in telegram dm(djeck_vorobey1) your email, I will share it with you)
3. If user has already opened position, please, show user this pop-up
4. To get data if user has opened position, call `/api/has-user-opened-position` this endpoint which will return:
```
{"has_opened_position": True or False}
```
5. If user click on `Close active position` redirect to `Dashboard` page
6. Pop-up view
Screenshot 2024-11-18 at 10 39 38

### PR DESCRIPTION
**Pull request by PoulavBhowmick03**
Fixes #213

### CODE REVIEW

File: frontend/src/pages/forms/Form.jsx [hunk 1]
```diff
@@ -28,13 +30,36 @@ const Form = ({ walletId, setWalletId }) => {
...
+        const data = await response.json();
+        setHasOpenPosition(data.has_opened_position);
+      } catch (error) {
+        console.error('Failed to check open position:', error);
+        setError('Failed to check position status');
+      }
+    };
+
+    checkOpenPosition();
+  }, [walletId]);
```
Comment (whateverfw) [inline frontend/src/pages/forms/Form.jsx:55]: api requests should be done using tanstack query library (that is already part of a project), also using axios, not fetch

File: frontend/src/components/openpositionpopup/popup.css
Comment (whateverfw) [inline frontend/src/components/openpositionpopup/popup.css:1]: use css variables for colors

File: frontend/src/components/openpositionpopup/StyledPopup.jsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
+/* eslint-disable react/no-unescaped-entities */
```
Comment (whateverfw) [inline frontend/src/components/openpositionpopup/StyledPopup.jsx:1]: avoid disabling eslint, fix issues instead

**Review by whateverfw — CHANGES_REQUESTED** [review]
please resolve merge conflict

**Comment by djeck1432** [comment]
@PoulavBhowmick03 Any updates?
please, sync with main

**Comment by PoulavBhowmick03** [comment]
Yes, give me 30 mins, updating the changes asked

**Pull request merged**